### PR TITLE
new section structure

### PR DESCRIPTION
**Pull request by MoritzLaurer**
# What does this PR do?

draft PR for testing new section structure discussed [internally here](https://huggingface.slack.com/archives/C06AMDKKDNF/p1716907801911879?thread_ts=1715954541.512919&cid=C06AMDKKDNF)

### CODE REVIEW

**Comment by MoritzLaurer** [comment]
PR is ready for review @stevhliu.
I'm not sure how to rename the overall documentation page to "Hugging Face Cookbook" (screenshot). 
Screenshot 2024-05-30 at 12 05 55

File: notebooks/en/_toctree.yml [hunk 1]
```diff
@@ -1,69 +1,77 @@
...
+          title: Embedding multimodal data for similarity search
+
+    - title: Agents Recipes
+      isExpanded: false
+      sections:
+        - local: agents
+          title: Build an agent with tool-calling superpowers using Transformers Agents
 
 - title: Enterprise Hub Cookbook
+  isExpanded: False
```
Comment (stevhliu) [inline notebooks/en/_toctree.yml:74]: I think it'd be better to leave this expanded so users can quickly click on the "Overview" to see how this is different from the open-source cookbook.

**Review by stevhliu — COMMENTED** [review]
Nice!

I think you'll have to change it [here](https://github.com/huggingface/moon-landing/blob/a30f18b9ab073b7d446c8e887a32165667064d8c/server/views/pages/DocsPage/consts.ts#L393) in the `title` field.

It may also be a good idea to update the `description` to include something about the Enterprise Hub or make it more general and not just "open-source".

Also cc'ing @merveenoyan for any feedback about the new structure since this would be a pretty big change 🙂

**Comment by MoritzLaurer** [comment]
I've opened [this internal PR](https://github.com/huggingface/moon-landing/pull/10262) to update the name and description of the cookbook. 

The present PR should be ready to be merged from my side @stevhliu @merveenoyan

**Review by merveenoyan — APPROVED** [review]

**Review by stevhliu — APPROVED** [review]
Thanks for reorganizing everything!

**Pull request merged**